charts/values.yaml

```diff
@@ -62,8 +64,12 @@ preUpgradeHook:
   ## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
   imagePullSecrets: []
   # - name: secretName
-  # -- Labels to be added to the pod hook job
-  podLabels: {}
+  # Labels to be added to the pod hook job
+  podLabels:
+    openebs.io/logging: "true"
+  # This helps in debugging by leaving the Job/Pod instances lying around
```

Comment:
You mean by default we'll leave the jobs around? or on failure only?

Reply:
We're not leaving the job around in success. In failure, we leave it around.
The comment here suggests that one could debug the pre-hook behaviour better by changing these values here, instead of having to tweak the template.